src/types/tcfv2/TCData.ts

```diff
@@ -88,15 +88,6 @@ export interface TCData {
 			consents: TCFv2ConsentList;
 			legitimateInterests: TCFv2ConsentList;
 		};
-		restrictions: {
-			[key: string]: {
-				/**
-				 * 0 - Not Allowed
-				 * 1 - Require Consents
-				 * 2 - Require Legitimate Interest
-				 */
```

Comment:
We’ve lost this comment! Worth turning into a type?

Reply:
oh no that was an oversight, restoring it